workspaces/telepath-js/README.md

```diff
@@ -12,12 +12,12 @@ Because both the browser and phone are likely to be behind distinct [NAT], we us
 
 Setting up a secure channel is done using these steps:
 
-1.  The web app requests a secure connection to the mobile app by invoking the `createChannel` function on the javascript library.
-2.  The `createChannel` function generates a random channel id `I` and a symmetric encryption key `E`.
-3.  The web app displays a QR code containing the channel id `I` and key `E`.
-4.  The owner of the phone opens the app, points the camera to the QR code.
-5.  The phone app extracts the channel id and the key from the QR code.
-6.  Both phone and web app can now communicate on channel `I`. They encrypt/decrypt their messages using key `E`.
+1. The web app requests a secure connection to the identity app by invoking the `createChannel` function on the javascript library.
```

Comment:
Since Telepath can be used without Cogito, it is better to talk about the 'mobile' app than about the 'identity' app.

Reply:
Hmm...I am using phrases "Cogito Identity App" and "Cogito Mobile App" interchangeably depending on the context. I do not't mind "identity app" as it is clear in this context what it means.